🔍 Validating Terraform configuration for 'dev'...
📝 Log: logs/validate_dev_20250712_180206.log

[0m[1mInitializing modules...[0m
[31m╷[0m[0m
[31m│[0m [0m[1m[31mError: [0m[0m[1mDuplicate resource "aws_lambda_function" configuration[0m
[31m│[0m [0m
[31m│[0m [0m[0m  on modules/lambda/processor_function.tf line 28:
[31m│[0m [0m  28: [4mresource "aws_lambda_function" "processor"[0m {[0m
[31m│[0m [0m
[31m│[0m [0mA aws_lambda_function resource named "processor" was already declared at
[31m│[0m [0mmodules/lambda/main.tf:30,1-43. Resource names must be unique per type in
[31m│[0m [0meach module.
[31m╵[0m[0m
[31m╷[0m[0m
[31m│[0m [0m[1m[31mError: [0m[0m[1mDuplicate resource "aws_lambda_function" configuration[0m
[31m│[0m [0m
[31m│[0m [0m[0m  on modules/lambda/router_function.tf line 28:
[31m│[0m [0m  28: [4mresource "aws_lambda_function" "router"[0m {[0m
[31m│[0m [0m
[31m│[0m [0mA aws_lambda_function resource named "router" was already declared at
[31m│[0m [0mmodules/lambda/main.tf:5,1-40. Resource names must be unique per type in
[31m│[0m [0meach module.
[31m╵[0m[0m
[31m╷[0m[0m
[31m│[0m [0m[1m[31mError: [0m[0m[1mDuplicate output definition[0m
[31m│[0m [0m
[31m│[0m [0m[0m  on modules/lambda/router_function.tf line 65:
[31m│[0m [0m  65: [4moutput "router_function_arn"[0m {[0m
[31m│[0m [0m
[31m│[0m [0mAn output named "router_function_arn" was already defined at
[31m│[0m [0mmodules/lambda/outputs.tf:1,1-29. Output names must be unique within a
[31m│[0m [0mmodule.
[31m╵[0m[0m
